tests/PluginTest.php

```diff
@@ -43,10 +42,14 @@ protected function setUp(): void
         }
 
         $extra = [
-            Plugin::EXTRA => [
-                Plugin::EXTRA_IGNORE_PATHS => [
-                    'IncompatibleSignature'
-                ]
+            'composer-attribute-collector' => [
+                'ignore-paths' => [
+                    'tests/Acme/PSR4/IncompatibleSignature',
+                    'CreateMenu', // should not match anything because it is not relative to project root
+                ],
+                'include-paths' => [
```

Comment:
Since the plugin includes everything by default, I'd rather have the option named as `only-paths`, because only these paths will be included.

Reply:
That's reasonable